zetaclient/orchestrator/bootstrap.go

```diff
@@ -247,7 +256,8 @@ func syncObserverMap(
 			mapSet[int64, interfaces.ChainObserver](observerMap, chainID, ob, onAfterAdd)
```

Comment:
Not related to the pr , but maybe we should move the mapping.go under pkg directory

Reply:
this is pretty internal for orchestrator only